mscore/inspector/inspector_hairpin.ui

```diff
@@ -293,6 +293,20 @@
        </property>
       </widget>
      </item>
+     <item row="2" column="0">
+      <widget class="QCheckBox" name="hairpinSingleNote">
+       <property name="text">
+        <string>Single note (de)crescendo</string>
+       </property>
+      </widget>
+     </item>
+     <item row="2" column="2">
+      <widget class="QToolButton" name="resetHairpinSingleNote">
+       <property name="text">
+        <string>...</string>
```

Comment:
also check the 'accessible name' for the reset button, compare with the others to see what I mean